client/src/components/OptionsMenu/OptionsMenu.component.tsx

```diff
@@ -275,6 +214,8 @@ const OptionsMenu = (): ReactElement => {
       </AccordionItem>
     </Accordion>
   )
+
+  return <Fragment>{accordionMenu}</Fragment>
```

Comment:
```suggestion
  return <>{accordionMenu}</>
```
This will also allow you to avoid the `<Fragment>` import, and thus minimise the amount of changes in the diff